packages/x-date-pickers/src/internals/hooks/usePicker/usePickerValue.ts

```diff
@@ -206,6 +206,13 @@ export interface UsePickerValueNonStaticProps<TValue>
    * Use in controlled mode (see open).
    */
   onOpen?: () => void;
+  /**
+   * If `true`, on dismiss will reset the value to the value when the picker was opened.
+   *
+   * Might make most sense when in need of resetting the value on modal dismiss.
+   * @default `true` for mobile, `false` for desktop.
```

Comment:
The default value of a boolean prop should be `false`, such that you end up with either

`<Components />` or `<Component resetValueOnDismiss />` but not `<Component resetValueOnDismiss={false} />`

Reply:
Thank you for clearing this thing up. 👍 
I've updated it, do you think it is now in a correct form? 🤔